locale/core.yml

```
@@ -151,6 +151,7 @@ core:
       delete_posts_forever_label: Delete posts forever
       delete_posts_label: Delete posts
       edit_posts_label: Edit posts
+      edit_user_label: Edit user
```

```suggestion
      edit_users_label: Edit users
```

To be consistent with the other labels, e.g. `delete_discussions_label`. Same change then needs to be made in the core PR, too.